### PR TITLE
Updated Readme

### PR DESCRIPTION
**Pull request by kevinpapst**
Changes proposed in this pull request:
- Moved contribution guide to https://github.com/kimai/kimai/blob/master/.github/CONTRIBUTING.md
- Added images
- Added feature list

Reasons for this pull request:
- Keep contribution guidelines in one place
- Animate users with some features 
- Brand the main github page with our images :-D 

See it in action here: https://github.com/kevinpapst/kimai/blob/readme/README.md

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -1,27 +1,21 @@
...
+
+## Quick start
 
 1. Fork Kimai or one of its [other repos][1]
 2. Clone repository to your local disc
-3. Create a branch (`git checkout -b my_kimai`)
-4. Commit your changes to your fork. In your commit message refer to the issue number if there is already one, e.g. (`git commit -am "[BUGFIX] short description of fix (resolves #4711)"`)
-5. Push to the branch (`git push origin my_kimai`)
+3. Checkout develop (`git checkout -b develop origin/develop`)
+3. Create a branch for your feature (`git checkout -b my_feature`)
```
Comment (simonschaufi) [inline .github/CONTRIBUTING.md]: if you add another item, you also need to increase the other items ;)

File: .github/CONTRIBUTING.md [hunk 2]
```diff
@@ -30,5 +24,22 @@ Here are a few rules to follow in order to ease code reviews, and discussions be
...
+Here are a few rules to follow in order to ease code reviews and discussions before maintainers accept and merge your work:
+
+* You SHOULD write documentation.
+* Please, write commit messages that make sense, and rebase your branch before submitting your Pull Request.
+* One may ask you to squash your commits too. This is used to "clean" your Pull Request before merging it (we don't want commits such as fix tests, fix 2, fix 3, etc.).
+* When creating your Pull Request on GitHub, you MUST write a description which gives the context and/or explains why you are creating it.
+
+## Important things to consider
+
+* Follow PSR-1 code style
```
Comment (simonschaufi) [inline .github/CONTRIBUTING.md]: PSR-2 or?

**Comment by kevinpapst** [comment]
Uups, PSR-2 for the win :-D
Accidentaly merged master, new PR?

**Comment by simonschaufi** [comment]
merge works fine

**Pull request merged**